src/nvim/ex_docmd.c

```diff
@@ -5311,6 +5332,87 @@ uc_check_code (
     break;
   }
 
+  case ct_MODS:
+  {
+    result = quote ? 2 : 0;
+    if (buf != NULL) {
+      if (quote) {
+        *buf++ = '"';
+      }
+      *buf = '\0';
+    }
+
+    bool multi_mods = false;
+
+    // :aboveleft and :leftabove
+    if (cmdmod.split & WSP_ABOVE) {
+      result += add_cmd_modifier(buf, "aboveleft", &multi_mods);
+    }
+    // :belowright and :rightbelow
+    if (cmdmod.split & WSP_BELOW) {
+      result += add_cmd_modifier(buf, "belowright", &multi_mods);
+    }
+    // :botright
+    if (cmdmod.split & WSP_BOT) {
+      result += add_cmd_modifier(buf, "botright", &multi_mods);
+    }
+
+    typedef struct {
+      bool *set;
```

Comment:
Also wondering about this rename.

Basically, try to thing about whether it is worth the cost of having to cross-reference different names. If there is no major gain, it's better to just use the original name, if it's not idea. If you rewrote the function and cross-referencing is irrelevant, _then_ it might make sense to rename things.